src/styles/watch.scss

```diff
@@ -33,7 +34,7 @@
 
   &__minute {
     position: absolute;
-    left: $main-size / 2 - $arrow-width / 2;
+    left: ($main-size / 2) - ($arrow-width / 2);
     top: 250px - $minute-height;
     height: $minute-height;
     width: $arrow-width;
```

Comment:
You use the same values ​​multiple times. ( `position: absolute;
    left: width: $arrow-width;`  ) use a placeholder for this.